test/testutils/testutils.cpp

```diff
@@ -27,6 +28,8 @@ namespace OCC {
 namespace TestUtils {
     TestUtilsPrivate::AccountStateRaii createDummyAccount()
     {
+        // ensure we have an instance of NetworkInformation
+        NetworkInformation::initialize();
```

Comment:
Should we move that to https://github.com/owncloud/client/blob/master/test/testutils/testutilsloader.cpp#L9